comp/core/status/statusimpl/status.go

```diff
@@ -193,17 +193,26 @@ func (s *statusImplementation) GetStatus(format string, verbose bool, excludeSec
 			}
 
 			if len(section) > 0 {
+				headerBuffer := new(bytes.Buffer)
+				sectionBuffer := new(bytes.Buffer)
 
 				for i, provider := range s.sortedProvidersBySection[section] {
+
 					if i == 0 {
```

Comment:
🥜 nitpick: ‏Since you only do that for first index you can put it above for loop ?

Reply:
hum tbh it was written like this before and I would prefer to avoid doing too much refacto on this file